app/webpacker/components/Tickets/TicketHeader/index.jsx

```diff
@@ -18,3 +22,16 @@ export default function TicketHeader({ ticketDetails, currentStakeholder, update
     </Card>
   );
 }
+
+function Heading({ ticketDetails }) {
+  const { ticket: { metadata_type: ticketType, metadata } } = ticketDetails;
+
+  switch (ticketType) {
+    case ticketTypes.edit_person:
+      return 'Edit Person';
```

Comment:
Why do `edit_person` tickets not have specific titles?
Or in other words, why is it okay to show `"Edit Person"` for this type, but not okay to just show `"Competition Results"` for the new `competition_results` type?

Reply:
For competition results, the title helps to identify which competition's ticket are we working on. Hence the competition name in title is necessary. While for edit person, WRT doesn't need any additional information in the title, so giving a generic title. We anyway can know the name of the requestor and other details from the workbench.

Reply:
Okay, then the follow-up question is: Why does Edit Person have all necessary information in the body, and Post Results doesn't?

It seems that for both ticket types (edit person and post results) there is some kind of "main" information or "identifying" information:
- Edit Person: The person being edited
- Post Results: The competition for which results are being posted

In one case, this "main" information appears in the header, and in the other case, it can appear in the body. That's very inconsistent, why?

Reply:
Okay, I understand that this is inconsistent. I revisited the edit person page and I felt that WCA ID is a necessary requirement. It's currently shown as part of `EditPersonForm`, but that form will soon be removed (I didn't discuss the alternative with you yet, I'll definitely discuss with you once I finish off the higher priority tasks).

For now, for consistency, I've added both generic heading as well as required information (as subheading). Hope this sounds good.